packages/connector-chrome/tests/events.ts

```diff
@@ -248,70 +248,63 @@ const updateLocalhost = (content, port) => {
     return transformed;
 };
 
-const testConnectorEvents = (connectorInfo) => {
-    const ConnectorConstructor: IConnectorConstructor = connectorInfo.ctor;
-    const name: string = connectorInfo.name;
-
-    test(`[${name}] Events`, async (t) => {
-        const { sonarwhal } = t.context;
-        const connector: IConnector = new ConnectorConstructor(sonarwhal, {});
-        const server = t.context.server;
+const name: string = 'chrome';
 
-        t.context.connector = connector;
+test(`[${name}] Events`, async (t) => {
+    const { sonarwhal } = t.context;
+    const connector: IConnector = new ChromeConnector(sonarwhal, {});
+    const server = t.context.server;
 
-        server.configure({
-            '/': updateLocalhost(fs.readFileSync(path.join(__dirname, './fixtures/common/index.html'), 'utf8'), server.port),
-            '/nellie.png': { content: fs.readFileSync(path.join(__dirname, './fixtures/common/nellie.png')) },
-            '/script.js': fs.readFileSync(path.join(__dirname, './fixtures/common/script.js'), 'utf8'),
-            '/script2.js': {
-                content: 'script.js',
-                status: 302
-            },
-            '/script3.js': {
-                content: 'script2.js',
-                status: 302
-            },
-            '/script4.js': {
-                content: 'script4.js',
-                status: 404
-            },
-            '/script5.js': null,
-            '/style.css': fs.readFileSync(path.join(__dirname, './fixtures/common/style.css'), 'utf8')
-        });
+    t.context.connector = connector;
 
-        const pendingEvents: any = events.map((event) => {
-            return updateLocalhost(event, server.port);
-        });
+    server.configure({
+        '/': updateLocalhost(fs.readFileSync(path.join(__dirname, './fixtures/common/index.html'), 'utf8'), server.port),
+        '/nellie.png': { content: fs.readFileSync(path.join(__dirname, './fixtures/common/nellie.png')) },
+        '/script.js': fs.readFileSync(path.join(__dirname, './fixtures/common/script.js'), 'utf8'),
```

Comment:
That file is empty, maybe drop this, and just use `content: ''` ?

(The same goes for the other files)